credit-card-recommendation/README.md

```diff
@@ -0,0 +1,26 @@
+# Credit card recommendation
+
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/InfuseAI/showcase/blob/main/credit-card-recommendation/notebook/primehub_sdk_credit_card_recommendation.ipynb)
+
+## Prerequisites
+
+1. PrimeHub EE 3.9.0 version to do the showcase.
```

Comment:
nit: since only one item, could revise to:
- PrimeHub v3.9.0 EE

Reply:
Done.